google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeployAction.java

```diff
@@ -221,18 +217,19 @@ public void run() {
     }
 
     private void stop(@NotNull UndeploymentTaskCallback callback) {
-      String moduleToStop;
+      DeployOutput deployOutput;
       try {
-        moduleToStop = parseDeployOutputToModule(deploymentOutput);
```

Comment:
SGTM. I'll move it to the constructor. The constructor will not throw any exception.